src/index.html

```diff
@@ -17,6 +30,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
```

Comment:
Ensure that the image source path `/src/images/logo.png` is correct and that the image exists at this location.